tests/baselines/reference/quickInfoInheritDoc.baseline

```diff
@@ -0,0 +1,414 @@
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "void",
+          "kind": "keyword"
+        }
+      ],
+      "documentation": [
+        {
+          "text": "BaseClass.func1",
+          "kind": "text"
+        }
+      ],
+      "tags": [
+        {
+          "name": "param",
+          "text": [
+            {
+              "text": "stuff1",
+              "kind": "parameterName"
+            },
+            {
+              "text": " ",
+              "kind": "space"
+            },
+            {
+              "text": "BaseClass.func1.stuff1",
+              "kind": "text"
+            }
+          ]
+        },
+        {
+          "name": "returns",
+          "text": [
+            {
+              "text": "BaseClass.func1.returns",
+              "kind": "text"
+            }
+          ]
+        },
+        {
+          "name": "inheritDoc"
+        },
+        {
+          "name": "param",
+          "text": [
+            {
+              "text": "stuff1",
+              "kind": "parameterName"
+            },
+            {
+              "text": " ",
+              "kind": "space"
+            },
+            {
+              "text": "SubClass.func1.stuff1",
+              "kind": "text"
+            }
+          ]
+        },
+        {
+          "name": "returns",
+          "text": [
+            {
+              "text": "SubClass.func1.returns",
+              "kind": "text"
+            }
+          ]
+        }
+      ]
+    }
+  },
+  {
+    "marker": {
+      "fileName": "/tests/cases/fourslash/quickInfoInheritDoc.ts",
+      "position": 1282,
+      "name": "3"
+    },
+    "quickInfo": {
+      "kind": "property",
+      "kindModifiers": "public,static",
+      "textSpan": {
+        "start": 1282,
+        "length": 12
+      },
+      "displayParts": [
+        {
+          "text": "(",
+          "kind": "punctuation"
+        },
+        {
+          "text": "property",
+          "kind": "text"
+        },
+        {
+          "text": ")",
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "SubClass",
+          "kind": "className"
+        },
+        {
+          "text": ".",
+          "kind": "punctuation"
+        },
+        {
+          "text": "someProperty",
+          "kind": "propertyName"
+        },
+        {
+          "text": ":",
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "string",
+          "kind": "keyword"
+        }
+      ],
+      "documentation": [
+        {
+          "text": "Applicable description always.",
+          "kind": "text"
+        },
+        {
+          "text": "\n",
+          "kind": "lineBreak"
+        },
+        {
+          "text": "text over tag",
```

Comment:
I thought `@inheritDoc` was supposed to insert the inherited text at the location of the tag. Here, it looks like it's getting prepended.

Reply:
It's appended in webstorm. 😆 
![bf84f3616e55995c069ab812743a75c](https://user-images.githubusercontent.com/6630042/163678165-35f40e33-c57b-4452-b979-4cd25803bd7c.png)